requirements.txt

```diff
@@ -2,4 +2,5 @@ arrow>=1.0.0
 click>=8.0
 click-didyoumean
 colorama; sys_platform == "win32"
+dateparser
```

Comment:
given that arrow is already here, wouldn't hurt to rework this change with `arrow.dehumanize(s)`.

Can't talk for the maintainers, but I'd say using existing dependencies rather than adding a new one would increase chance of this being pulled in.

Reply:
This also looks like it could replace my PR more cleanly, so if you were to ingest the tests that my PR supports and take it further to days/weeks etc, that'd be an improvement.